asistente_ladm_col/tests/test_change_detections_supplies.py

```diff
@@ -0,0 +1,1189 @@
+import nose2
+
+from qgis.core import (QgsWkbTypes,
+                       NULL)
+from qgis.testing import (start_app,
+                          unittest)
+
+start_app()  # need to start before asistente_ladm_col.tests.utils
+
+from asistente_ladm_col.utils.qgis_utils import QGISUtils
+from asistente_ladm_col.logic.ladm_col.data.ladm_data import LADM_DATA
+from asistente_ladm_col.config.general_config import LAYER
+from asistente_ladm_col.tests.utils import (get_pg_conn,
+                                            normalize_responce,
+                                            restore_schema)
+
+
+class TestChangeDetectionsSupplies(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        print("INFO: Restoring databases to be used")
+        restore_schema('test_change_detections')
+        cls.db_pg = get_pg_conn('test_change_detections')
+        result = cls.db_pg.test_connection()
+        print('test_connection', result)
+
+        if not result[1]:
+            print('The test connection is not working')
+            return
+
+        cls.qgis_utils = QGISUtils()
+        cls.ladm_data = LADM_DATA(cls.qgis_utils)
+
+    def test_get_plots_related_to_parcels_supplies(self):
+        print("\nINFO: Validating get plots related to parcels in supplies model (Case: t_id)...")
+
+        result = self.db_pg.test_connection()
+        self.assertTrue(result[0], 'The test connection is not working')
+        self.assertIsNotNone(self.db_pg.names.OP_BOUNDARY_POINT_T, 'Names is None')
+
+        parcel_ids_tests = [list(), [1000], [1000, 1001, 1002]]
+        plot_ids_tests = [list(), [1112], [1112, 1102, 1086]]
+
+        count = 0
+        for parcel_ids_test in parcel_ids_tests:
+            plot_ids = self.ladm_data.get_plots_related_to_parcels_supplies(self.db_pg, parcel_ids_test, self.db_pg.names.T_ID_F)
+            # We use assertCountEqual to compare if two lists are the same regardless of the order of their elements.
+            # https://docs.python.org/3.2/library/unittest.html#unittest.TestCase.assertCountEqual
+            self.assertEqual(sorted(plot_ids), sorted(plot_ids_tests[count]), "Failure with data set {}".format(count + 1))
```

Comment:
> We use assertCountEqual

We didn't, right?

Reply:
Remove comment.

Reply:
Unnecessary comments were removed, thanks.